smart-contracts/contracts/BridgeProxy/BridgeProxy.sol

```diff
@@ -72,4 +72,7 @@ contract BridgeProxy {
     }
     _;
   }
+
+  // Bridge contracts should be able to receive ETH
+  receive() external payable {}
```

Comment:
Well, I am not sure we need this for all.. for example CCTP does not... which is why I think it is better to keep it at the `[OPStackNativeBridgeProxy.sol](https://github.com/relayprotocol/relay-protocol/pull/245/files#diff-fd3bba90c592c8034c1a598e13f4820576883d9465ead2e0a9da8163e6ed400d)` level!